aws/logs_monitoring/tools/integration_tests/docker-compose.yml

```diff
@@ -6,6 +6,11 @@ services:
     command: lambda_function.lambda_handler
     environment:
       AWS_ACCOUNT_ID: 0000000000
+      AWS_ACCESS_KEY_ID: "${AWS_ACCESS_KEY_ID}"
+      AWS_SECRET_ACCESS_KEY: "${AWS_SECRET_ACCESS_KEY}"
+      AWS_SECRET_ACCESS_KEY: "${AWS_SECRET_ACCESS_KEY}"
```

Comment:
Need to set `AWS_SECRET_ACCESS_KEY` twice?

Reply:
Good catch, removing the dup.